themes/src/main/resources/theme/keycloak.v2/account/messages/messages_en.properties

```diff
@@ -89,6 +89,9 @@ password-help-text=Log in by entering your password.
 password=My Password
 otp-display-name=Authenticator Application
 otp-help-text=Enter a verification code from authenticator application.
+recovery-authn-code=My Recovery Authentication Codes
+recovery-authn-codes-display-name=Recovery Authentication Codes
+recovery-authn-codes-help-text=Recovery Authentication Codes can be used as a second factor or to recover access to your account in the event of a lost second factor.
```

Comment:
This should be made a bit shorter

Reply:
This sounds better -> "These codes can be used to regain your access in case your other 2FA means are not available"

Reply:
+1